e2e/process-services-cloud/form-field/visibility-condition.e2e.ts

```diff
@@ -242,4 +250,79 @@ describe('Visibility conditions - cloud', () => {
         await widget.textWidget().setValue(widgets.textThreeId, 'something');
         await widget.textWidget().isWidgetVisible(widgets.textOneId);
     });
+
+    it('[C309867] Display value widget displays the value of a form variable', async () => {
```

Comment:
Incorrect test name format

Reply:
Corrected the name & updated in TestRail as well.